ci/testnet-sanity.sh

```diff
@@ -44,23 +49,22 @@ else
   )
 fi
 
-# TODO: sanity test with validator
-# echo "--- $NET_URL: validator sanity"
-# (
-#   export USE_SNAP=1
-#   ./multinode-demo/setup.sh -t validator
-#   set -e pipefail
-#   timeout 10s ./multinode-demo/validator.sh "$NET_URL" 2>&1 | tee log
-# )
-#
-# (
-#   set +e
-#   panic=$(timeout 10s tail -f /var/snap/solana/current/validator/current | grep -C100 panic)
-#
-#   if [[ -n $panic ]]; then
-#       echo Panic observed: "$panic"
-#       exit 1
-#   fi
-# )
+echo "--- $NET_URL: validator sanity"
+if [[ -z $NO_VALIDATOR_SANITY ]]; then
+  (
+    ./multinode-demo/setup.sh -t validator
+    set -e pipefail
+    timeout 10s ./multinode-demo/validator.sh "$NET_URL" 2>&1 | tee validator.log
+  )
+  wc -l validator.log
+  if grep -C100 panic validator.log; then
+    echo "^^^ +++ Panic observed"
+    exit 1
+  else
+    echo "Validator log looks ok"
+  fi
+else
+  echo "^^^ +++ Validator sanity disabled (NO_VALIDATOR_SANITY defined)"
+fi
```

Comment:
just out of curiosity, could we perhaps do this on actual validator nodes?

Reply:
Yeah we could totally do that.  It would require ssh-ing into each validator node.  `testnet-deploy.sh` already does similar stuff so it's just legwork to make that happen